app/components/global-navigation.js

```diff
@@ -173,6 +173,11 @@ export default Component.extend({
 			this.onSearchSuggestionChosen(suggestion);
 		},
 
+		onSearchSuggestionsImpression(suggestions, searchId) {
+			// eslint-disable-next-line no-console
+			console.info('Suggestions impression', suggestions, 'with searchId', searchId);
```

Comment:
Why we need console.log here?

Reply:
This is definitely not the place for console.log, it was supposed to land in a controller of the dummy app :)